cmd/utils.go

```diff
@@ -965,3 +966,18 @@ func auditLogInternal(ctx context.Context, bucket, object string, opts AuditLogO
 	ctx = logger.SetAuditEntry(ctx, &entry)
 	logger.AuditLog(ctx, nil, nil, nil)
 }
+
+// hashStr2Int - Returns an integer hash of the given string
+func hashStr2Int(s string) uint32 {
+	h := fnv.New32a()
+	h.Write([]byte(s))
+	return h.Sum32()
+}
+
+// mapIfNotPresent - Maps the given key to the value only if the key is not present in the map
+func mapIfNotPresent(m map[string]string, key string, val string) {
+	_, found := m[key]
+	if !found {
+		m[key] = val
+	}
+}
```

Comment:
Is this really required?

Reply:
At the very least, just move it into `anonymizeHost` as a private func.

Reply:
Will do